docs/src/manual/solutions.md

```diff
@@ -3,7 +3,7 @@ CurrentModule = JuMP
 DocTestSetup = quote
     using JuMP, GLPK
 end
-DocTestFilters = [r"≤|<=", r"≥|>=", r" == | = ", r" ∈ | in ", r"MathOptInterface|MOI"]
+DocTestFilters = [r"≤|<=", r"≥|>=", r" == | = ", r" ∈ | in ", r"MathOptInterface|MOI", r"[0-9]+.[0-9]+"]
```

Comment:
We don't want to add this here, because it will now ignore _every_ number in this file. This could lead to incorrect results slipping past the tests. It should go
````
```jldoctest solutions; filter=r"[0-9]+.[0-9]+"
````

Reply:
Oh. sorry. I should be. I fixed it.